packages/mongodb-chatbot-ui/src/DevCenterChatbot.tsx

```diff
@@ -40,6 +40,7 @@ export function DevCenterChatbot(props: DevCenterChatbotProps) {
         />
       </>
     ),
+    shouldUseHotkey: true,
```

Comment:
I know we aren't using this component, but I set this to `true` here so it would show up in the tester